### ISSUE TITLE
interface_ppi_confidence.ipynb, missing data and incorrect module calling

### ISSUE THREAD

**Issue by rubenalv** [LBM-EPFL/PeSTo#5]
Modules are not called according to the repository structure, eg:
`from model import Model`
should be
`from model.model import Model`

Also, example data missing:
`dataset = Dataset("datasets/contacts_rr5A_64nn_8192.h5")`

I could suggest to rerun all .ipynb files in a fresh conda enviroment with a cloned git, to ensure they work?

**Comment by mikewzp** [comment]
@rubenalv Yes, I also encounter the same issue when run `interface_ppi_benchmark.ipynb`. How do you solve the missing h5 file?

**Comment by rubenalv** [comment]
@yzhang-github-pub At the moment I am setting up the tools, but have not done much analysis. They suggest a probability cutoff in the paper, but that would depend on your goals. I thought of using stretches of aminoacids (eg minimum of 3), but then what happens if the PPI is determined by adjacent but not contiguous aminoacids, would it be better to look at proximity of coordinates of residues of high probability?
I also noticed that the results depend on what you input: in the example in the Pesto website (https://pesto.epfl.ch/) you can choose a single chain or all chains, and the PPI probabilities are very, very different depending on your choice. Proteins can change conformation upon binding to each other, so it makes sense, but the observation makes me be cautious about conclusions about presence/lack of PPI in a prediction.

**Comment by rubenalv** [comment]
@yzhang-github-pub, I agree. The paper is well written, but the github is not really open to the public and would greatly benefit from annotation. That would help increase the citations to the paper...
If it helps, this is the structure of a working solution:

===========
LICENSE  README.md  run_pesto_model.py

examples/
examples_in  examples_out

model/
config.py  data_handler.py  main.py  model_ckpt_i_v4_1_2021-09-07_11-21.pt  model_i_v4_1_2021-09-07_11-21.pt  model.py  model_version.txt  __pycache__

src/
data_encoding.py  dataset.py  logger.py  model_operations.py  __pycache__  scoring.py  structure_io.py  structure.py
===========


Beware of some of the files in src/, I recall that one appeared elsewhere with the same name but had slightly different code. I think it was not a big deal, just that it wrote the last two columns of the output pdbs as 0,b-factor or b-factor,b-factor... but just so you know if you see an inconsistency of outputs.

The script run_pesto_model.py is customized, inspired from the jupyter notebooks, and I uploaded the code in a comment above https://github.com/LBM-EPFL/PeSTo/issues/5#issuecomment-1672888622

**Comment by rubenalv** [comment]
In case someone reads this thread, I decided to do a clean up of the repository to make a working version - feel free to use my PeSTo fork.
